handler/src/main/java/io/netty/handler/ssl/OpenSslCertificateException.java

```diff
@@ -24,6 +27,357 @@
  * SSL Record. This only work when {@link SslProvider#OPENSSL} is used.
  */
 public final class OpenSslCertificateException extends CertificateException {
+    public static final int X509_V_OK;
+    public static final int X509_V_ERR_UNSPECIFIED;
+    public static final int X509_V_ERR_UNABLE_TO_GET_ISSUER_CERT;
+    public static final int X509_V_ERR_UNABLE_TO_GET_CRL;
+    public static final int X509_V_ERR_UNABLE_TO_DECRYPT_CERT_SIGNATURE;
+    public static final int X509_V_ERR_UNABLE_TO_DECRYPT_CRL_SIGNATURE;
+    public static final int X509_V_ERR_UNABLE_TO_DECODE_ISSUER_PUBLIC_KEY;
+    public static final int X509_V_ERR_CERT_SIGNATURE_FAILURE;
+    public static final int X509_V_ERR_CRL_SIGNATURE_FAILURE;
+    public static final int X509_V_ERR_CERT_NOT_YET_VALID;
+    public static final int X509_V_ERR_CERT_HAS_EXPIRED;
+    public static final int X509_V_ERR_CRL_NOT_YET_VALID;
+    public static final int X509_V_ERR_CRL_HAS_EXPIRED;
+    public static final int X509_V_ERR_ERROR_IN_CERT_NOT_BEFORE_FIELD;
+    public static final int X509_V_ERR_ERROR_IN_CERT_NOT_AFTER_FIELD;
+    public static final int X509_V_ERR_ERROR_IN_CRL_LAST_UPDATE_FIELD;
+    public static final int X509_V_ERR_ERROR_IN_CRL_NEXT_UPDATE_FIELD;
+    public static final int X509_V_ERR_OUT_OF_MEM;
+    public static final int X509_V_ERR_DEPTH_ZERO_SELF_SIGNED_CERT;
+    public static final int X509_V_ERR_SELF_SIGNED_CERT_IN_CHAIN;
+    public static final int X509_V_ERR_UNABLE_TO_GET_ISSUER_CERT_LOCALLY;
+    public static final int X509_V_ERR_UNABLE_TO_VERIFY_LEAF_SIGNATURE;
+    public static final int X509_V_ERR_CERT_CHAIN_TOO_LONG;
+    public static final int X509_V_ERR_CERT_REVOKED;
+    public static final int X509_V_ERR_INVALID_CA;
+    public static final int X509_V_ERR_PATH_LENGTH_EXCEEDED;
+    public static final int X509_V_ERR_INVALID_PURPOSE;
+    public static final int X509_V_ERR_CERT_UNTRUSTED;
+    public static final int X509_V_ERR_CERT_REJECTED;
+    public static final int X509_V_ERR_SUBJECT_ISSUER_MISMATCH;
+    public static final int X509_V_ERR_AKID_SKID_MISMATCH;
+    public static final int X509_V_ERR_AKID_ISSUER_SERIAL_MISMATCH;
+    public static final int X509_V_ERR_KEYUSAGE_NO_CERTSIGN;
+    public static final int X509_V_ERR_UNABLE_TO_GET_CRL_ISSUER;
+    public static final int X509_V_ERR_UNHANDLED_CRITICAL_EXTENSION;
+    public static final int X509_V_ERR_KEYUSAGE_NO_CRL_SIGN;
+    public static final int X509_V_ERR_UNHANDLED_CRITICAL_CRL_EXTENSION;
+    public static final int X509_V_ERR_INVALID_NON_CA;;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty statement. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AEmptyStatementUsageCheck)